recipes/iir1/all/conanfile.py

```diff
@@ -72,6 +76,7 @@ def _configure_cmake(self):
             return self._cmake
 
         self._cmake = CMake(self)
+        self._cmake.definitions['IIR1_NO_EXCEPTIONS'] = self.options.noexceptions
```

Comment:
```suggestion
        self._cmake.definitions['IIR1_NO_EXCEPTIONS'] = self.options.get_safe("noexceptions", False)
```
You still can use for any version. CMake will show a warning as unused definition, but it does not matter.